<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Reinhold Kainhofer &lt;reinhold@kainhofer.com&gt;</author>
 <comment>Configuration page for the print day mode.</comment>
 <class>CalPrintTodoConfig_Base</class>
 <widget class="QWidget" name="CalPrintTodoConfig_Base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>566</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Print to-dos options:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="mTitleLabel">
       <property name="text">
        <string>&amp;Title:</string>
       </property>
       <property name="buddy">
        <cstring>mTitle</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="mTitle">
       <property name="text">
        <string>To-do List</string>
       </property>
        <property name="toolTip">
         <string>Name for this to-do list</string>
        </property>
        <property name="whatsThis">
         <string>Enter a name for this to-do list that will be put at the top of the print-out</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="mPrintType">
     <property name="title">
      <string>To-dos to Print</string>
     </property>
     <layout class="QGridLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="mPrintAll">
        <property name="text">
         <string>Print &amp;all to-dos</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="mPrintUnfinished">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Print &amp;unfinished to-dos only</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="mPrintDueRange">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Print only to-dos due in the &amp;range:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="mFromDateLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Start date:</string>
          </property>
          <property name="buddy">
           <cstring>mFromDate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KDateComboBox" name="mFromDate">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="whatsThis">
           <string>If you want to print more days at once, you can define a range of dates with this option and the &lt;i&gt;End date&lt;/i&gt; option. This option is used to define the start date.</string>
          </property>
          <item>
           <property name="text">
            <string>2009-01-19</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mToDateLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;End date:</string>
          </property>
          <property name="buddy">
           <cstring>mToDate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KDateComboBox" name="mToDate">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="whatsThis">
           <string>If you want to print more days at once, you can define a range of dates with this option and the &lt;i&gt;Start date&lt;/i&gt; option. This option is used to define the end date.</string>
          </property>
          <item>
           <property name="text">
            <string>2009-01-19</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>128</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="mSecurity">
     <property name="title">
      <string>Security Exclusions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="mExcludeConfidential">
        <property name="whatsThis">
         <string>Check this option to exclude confidential events.</string>
        </property>
        <property name="text">
         <string>Exclude c&amp;onfidential</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mExcludePrivate">
        <property name="whatsThis">
         <string>Check this option to exclude private events</string>
        </property>
        <property name="text">
         <string>Exclude pri&amp;vate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="includeInfoBox">
     <property name="title">
      <string>Include Information</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="mDescription">
        <property name="text">
         <string>Include &amp;descriptions</string>
        </property>
        <property name="toolTip">
         <string>Print item descriptions</string>
        </property>
        <property name="whatsThis">
         <string>Check this option if you want to see the item descriptions printed.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mPriority">
        <property name="text">
         <string>&amp;Priority</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mDueDate">
        <property name="text">
         <string>Due date</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mPercentComplete">
        <property name="text">
         <string>Per&amp;centage completed</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>mDescription</zorder>
     <zorder>mDueDate</zorder>
     <zorder>mPercentComplete</zorder>
     <zorder>mPriority</zorder>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="sortingOptionsBox">
     <property name="title">
      <string>Sorting Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="sortFieldLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Sort field:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="mSortField">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="sortDirectionLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Sort direction:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="mSortDirection">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QGroupBox" name="mGeneralGroup">
     <property name="title">
      <string comment="@title general print settings">General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="mConnectSubTodos">
        <property name="text">
         <string>Co&amp;nnect sub-to-dos with its parent</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mStrikeOutCompleted">
        <property name="text">
         <string>Strike &amp;out completed to-do summaries</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mPrintFooter">
        <property name="text">
         <string>Print &amp;Footer</string>
        </property>
        <property name="toolTip">
         <string>Print a datetime footer on each page</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to print a small footer on each page that contains the date of the print.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDateComboBox</class>
   <extends>KComboBox</extends>
   <header>kdatecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mTitle</tabstop>
  <tabstop>mPrintAll</tabstop>
  <tabstop>mPrintUnfinished</tabstop>
  <tabstop>mPrintDueRange</tabstop>
  <tabstop>mFromDate</tabstop>
  <tabstop>mToDate</tabstop>
  <tabstop>mDescription</tabstop>
  <tabstop>mPriority</tabstop>
  <tabstop>mDueDate</tabstop>
  <tabstop>mPercentComplete</tabstop>
  <tabstop>mConnectSubTodos</tabstop>
  <tabstop>mStrikeOutCompleted</tabstop>
  <tabstop>mSortField</tabstop>
  <tabstop>mSortDirection</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mPrintDueRange</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFromDateLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mPrintDueRange</sender>
   <signal>toggled(bool)</signal>
   <receiver>mToDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mPrintDueRange</sender>
   <signal>toggled(bool)</signal>
   <receiver>mToDateLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mPrintDueRange</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFromDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
